src/objects/jolt_shaped_object_impl_3d.cpp

```diff
@@ -345,6 +345,18 @@ void JoltShapedObjectImpl3D::post_step(float p_step, JPH::Body& p_jolt_body) {
 	previous_jolt_shape = nullptr;
 }
 
+bool JoltShapedObjectImpl3D::_is_huge() const {
+	for (const JoltShapeInstance3D& shape : shapes) {
+		if (shape.is_enabled() &&
+			shape.get_shape()->get_type() == PhysicsServer3D::SHAPE_WORLD_BOUNDARY)
+		{
+			return true;
+		}
+	}
+
+	return false;
+}
```

Comment:
@jrouwe Would you say it makes sense to generalize this method into calculating the AABB for the body as a whole (while caching the ones from the complex shapes) and using some arbitrary size as the cut-off to determine whether a body is "huge"?

That way you wouldn't only have `PlaneShape` bodies in this new broadphase layer, but also whatever other "huge" static body that the user decided to add to their world, which might perhaps speed things up a little bit in the regular `BODY_STATIC` broadphase layer?

I'm thinking there would only ever really be a handful of bodies in this new `BODY_STATIC_SLOW` layer either way, say maybe 6x `PlaneShape` and some huge `MeshShape` or two at most.

Reply:
Yes, I think that makes sense.